gems/pending/Gemfile

```diff
@@ -21,7 +21,7 @@ gem "excon",                   "~>0.40",            :require => false
 gem "ezcrypto",                "=0.7",              :require => false
 gem "ffi",                     "~>1.9.3",           :require => false
 gem "ffi-vix_disk_lib",        "~>1.0.2",           :require => false  # used by lib/VixDiskLib
-gem "fog",                     "~>1.37.0",          :require => false
+gem "fog-openstack",           "~>0.1.2",           :require => false
 gem "fog-core",                "!=1.31.1",          :require => false
```

Comment:
Thanks for trying this out. Do you think we can drop fog-core also? Both [fog-openstack](https://github.com/fog/fog-openstack/blob/master/fog-openstack.gemspec#L22) and [fog-google](https://github.com/fog/fog-google/blob/master/fog-google.gemspec#L25) pull it in.  Seems like one less dependency restriction someone might trip over down the road.